setup.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import setuptools
+
+with open("README.md", "r", encoding="utf-8") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="googleapis-storage-testbench",
+    version="0.4.0",
+    author="Google LLC",
+    author_email="googleapis-packages@google.com",
+    description="A testbench for Google Cloud Storage client libraries",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/googleapis/storage-testbench",
+    project_urls={
+        "Bug Tracker": "https://github.com/googleapis/storage-testbench/issues",
+    },
+    classifiers=[
+        "Programming Language :: Python :: 3",
+        "License :: OSI Approved :: Apache-2.0",
+        "Operating System :: OS Independent",
+    ],
+    package_dir={"": "."},
```

Comment:
Perhaps we don't need to declare the optional `package_dir` since we're following a common convention?

Reply:
SGTM.  Thanks.


Any thoughts on the package name (`googleapis-storage-testbench`) and the fact that it creates a `testbench` module?

It also creates  a `gcs` and `google/storage/v2` modules, but we will remove those eventually.

Reply:
^^ I don't think that would be a problem. I've been looking, comparing/contrasting with other googleapis-python repositories, and it's done similarly 😬

Once we change the structure, `/testbench` will kind of be the overarching directory, then we might want to set `package_dir`. Though it doesn't seem necessary since it's relatively easy to declare the packages.